mmtrack/models/sot/siamrpn.py

```diff
@@ -245,6 +248,98 @@ def track(self, img, bbox, z_feat, avg_channel):
         best_bbox = self._bbox_clip(best_bbox, img.shape[2], img.shape[3])
         return best_score, best_bbox
 
+    def simple_test_vot(self, img, frame_id, gt_bboxes, img_metas=None):
+        """Test using VOT test mode.
+
+        Args:
+            img (Tensor): of shape (1, C, H, W) encoding input image.
+            frame_id (int): the id of current frame in the video.
+            gt_bboxes (list[Tensor]): list of ground truth bboxes for each
+                image with shape (1, 4) in [tl_x, tl_y, br_x, br_y] format or
+                shape (1, 8) in [x1, y1, x2, y2, x3, y3, x4, y4].
+            image_shape (tuple | list): used as the bounds when calculating
+                overlap.
+
+        Returns:
+            bbox_pred (Tensor): in [tl_x, tl_y, br_x, br_y] format.
+            best_score (Tensor): the tracking bbox confidence in range [0,1],
+                and the score of initial frame is -1.
+        """
+        if frame_id == 0:
+            self.init_frame_id = 0
+        if self.init_frame_id == frame_id:
+            # initialization
+            gt_bboxes = gt_bboxes[0][0]
+            self.memo = Dict()
+            self.memo.bbox = quad2bbox(gt_bboxes)
+            self.memo.z_feat, self.memo.avg_channel = self.init(
+                img, self.memo.bbox)
+            # 1 denotes the initialization state
+            bbox_pred = img.new_tensor([1.])
+            best_score = -1.
+        elif self.init_frame_id > frame_id:
+            # 0 denotes unknown state, namely the skipping frame after failure
+            bbox_pred = img.new_tensor([0.])
+            best_score = -1.
+        else:
+            # normal tracking state
+            best_score, self.memo.bbox = self.track(img, self.memo.bbox,
+                                                    self.memo.z_feat,
+                                                    self.memo.avg_channel)
+            # convert bbox to region
+            track_bbox = bbox_cxcywh_to_x1y1wh(self.memo.bbox).cpu().numpy()
+            track_region = bbox2region(track_bbox)
+            gt_bbox = gt_bboxes[0][0].cpu().numpy()
```

Comment:
use `bbox_xyxy_to_x1y1wh` to change `gt_bbox` when the shape of `gt_bbox` is `(4, )`